### DISCUSSION TITLE
Delayed op use cases out of date?

### DISCUSSION THREAD

**Discussion by billpratt** [cloud-custodian/cloud-custodian#7329] [General]
The following AWS use cases show examples of delayed operations using custom tags. When implementing these types of policies for Azure, I noticed you have to add the custom tag to the ```marked-for-op``` type. Is this the case for AWS?

http://capitalone.github.io/cloud-custodian/docs/usecases/rdsdeleteunused.html
http://capitalone.github.io/cloud-custodian/docs/usecases/elbgarbagecollection.html

**Comment by kapilt** [comment]
there is a default, the example just shows overriding it, if you have multiple workflows running against the same resource type, its important to use a separate tag per workflow to avoid them overriding each other's state.

**Comment by billpratt** [comment]
In [elbgarbagecollection policy](http://capitalone.github.io/cloud-custodian/docs/usecases/elbgarbagecollection.html) you are using tag  ```tag: maid_status``` The tag appears to be missing in filter step for deletion
```
filters:
     - type: marked-for-op
       op: delete
actions:
```

**Comment by kapilt** [comment]
yeah.. `maid_status` is the default value here (set in c7n/tags.py DEFAULT_TAG). prior to being public, custodian had a different name, which is still reflected in this value and the cloud watch metrics namespace (CloudMaid), we ended up preserving those due to our compatibility guarantees.. and its kept going. probably worth updating the examples to always be explicit or at least document the default.
